elected_targets/admin.py

```diff
@@ -59,3 +60,56 @@ class SenatememAdmin(admin.ModelAdmin):
     search_fields = ['first', 'last']
 
 admin.site.register(Senatemem, SenatememAdmin)
+
+
+class StatesenatememAdmin(admin.ModelAdmin):
+    readonly_fields = [
+        'state', 'prefix', 'suffix', 'phone', 'fax',
+        'suite', 'leadership', 'plus4', 'web'
+    ]
+    fields = [
+        'first', 'last', 'email', 'nickname', 'gender', 'party',
+        'state', 'prefix', 'suffix', 'phone', 'fax',
+        'suite', 'leadership', 'plus4', 'web'
+    ]
+    list_display = ('state', 'seat', 'first', 'last')
+    list_filter = (('state', DropdownFilter),) if DropdownFilter else ['state']
+    list_sort = ['state', 'seat', 'first', 'last']
+    search_fields = ['first', 'last', 'seat']
+
+admin.site.register(Statesenatemem, StatesenatememAdmin)
+
+
+class StatehousememAdmin(admin.ModelAdmin):
+    readonly_fields = [
+        'state', 'prefix', 'suffix', 'phone', 'fax',
+        'suite', 'leadership', 'plus4', 'web'
+    ]
+    fields = [
+        'first', 'last', 'email', 'nickname', 'gender', 'party',
+        'state', 'prefix', 'suffix', 'phone', 'fax',
+        'suite', 'leadership', 'plus4', 'web'
+    ]
+    list_display = ('state', 'district', 'first', 'last')
+    list_filter = (('state', DropdownFilter),) if DropdownFilter else ['state']
+    list_sort = ['state', 'district', 'first', 'last']
+    search_fields = ['first', 'last', 'district']
+
+admin.site.register(Statehousemem, StatehousememAdmin)
+
+
+class GovernorAdmin(admin.ModelAdmin):
+    readonly_fields = [
+        'state', 'phone', 'fax'
+    ]
+    fields = [
+        'first', 'last', 'email', 'gender', 'party','state', 'phone', 'fax'
```

Comment:
nit: missing space after party (maybe run pep8 on the file)

Also, can we add facebook, twitter fields, too?

Reply:
It looks like only the Governor model has fb and tw fields, so i added them there.